### PR TITLE
Feature/559

### PR DESCRIPTION
**Pull request by M1ndForce**
#559

### CODE REVIEW

File: src/configs/index.js [hunk 1]
```diff
@@ -359,19 +343,7 @@ export const commentFields = {
...
-  'phoneNumber',
-  'country',
-  'region',
-  'city',
-  'street',
-  'buildingNumber',
-  'appartment',
-  'zipcode'
-];
+export const profileFields = ['firstName', 'lastName', 'email', 'phoneNumber', 'country', 'region', 'city', 'street', 'buildingNumber', 'appartment', 'zipcode'];
```
Comment (AndriyLymych) [inline src/configs/index.js:346]: pretify this line

File: src/pages/product-list-page/product-list-filter/price-filter/price-filter.js [hunk 2]
```diff
@@ -21,12 +18,14 @@ const PriceFilter = () => {
   const searchParams = new URLSearchParams(search);
   const { priceFilter, page, defaultPage } = URL_QUERIES_NAME;
 
-  const { filters, language } = useSelector(
-    ({ Products, Language, Currency }) => ({
-      filters: Products.filters.priceFilter,
-      language: Language.language
-    })
-  );
+  const { filters, language, currency, maxPrice, minPrice } = useSelector(({ Products, Language, Currency }) => ({
```
Comment (AndriyLymych) [inline src/pages/product-list-page/product-list-filter/price-filter/price-filter.js:21]: same

File: src/pages/product-list-page/product-list-filter/price-filter/price-filter.js [hunk 3]
```diff
@@ -48,12 +49,12 @@ const PriceFilter = () => {
...
   };
+  const min = minPrice ? minPrice[currency].value / 100 : 0;
+  const max = maxPrice ? maxPrice[currency].value / 100 : 1000;
   return (
     <FormGroup data-cy='price_filter'>
       <Typography id='range-slider' gutterBottom>
-        {PRICE_TEXT[language].value}: {PRICE_FROM[language].value}{' '}
-        {Math.round(filters[0] / 100)}- {PRICE_TO[language].value}{' '}
-        {Math.round(filters[1] / 100)}
+        {PRICE_TEXT[language].value}: {PRICE_FROM[language].value} {Math.round(filters[0] / 100)}- {PRICE_TO[language].value} {Math.round(filters[1] / 100)}
```
Comment (AndriyLymych) [inline src/pages/product-list-page/product-list-filter/price-filter/price-filter.js:57]: same

File: src/redux/products/products.operations.js [hunk 4]
```diff
@@ -95,23 +101,11 @@ const getFilteredProducts = async ({ state, currency }) => {
...
-          filter: {
-            colors: $colors
-            pattern: $patterns
-            price: $price
-            category: $category
-            isHotItem: $isHotItem
-            models: $models
-            currency: $currency
-          }
+          filter: { colors: $colors, pattern: $patterns, price: $price, category: $category, isHotItem: $isHotItem, models: $models, currency: $currency }
```
Comment (AndriyLymych) [inline src/redux/products/products.operations.js:104]: same

**Comment by Kiyooooko** [comment]
you should write in console:  git pull origin development
IMAGE 2021-02-22 13:07:30

File: src/pages/product-list-page/product-list-filter/price-filter/price-filter.js (hunk 3 above)
Comment (M1ndForce) [reply src/pages/product-list-page/product-list-filter/price-filter/price-filter.js:57]: frixed

File: src/redux/products/products.operations.js (hunk 4 above)
Comment (M1ndForce) [reply src/redux/products/products.operations.js:104]: fixed

File: src/pages/product-list-page/product-list-filter/product-list-filter.js [hunk 5]
```diff
@@ -27,10 +27,9 @@ const ProductListFilter = () => {
   const handleClearFilter = () => {
     const sortQuery = searchParams.get(sort);
     const quantityPerPage = searchParams.get(countPerPage);
-    history.push(
-      `/products/?page=1&sort=${sortQuery}&countPerPage=${quantityPerPage}`
-    );
+    history.push(`/products/?page=1&sort=${sortQuery}&countPerPage=${quantityPerPage}`);
```
Comment (AndriyLymych) [inline src/pages/product-list-page/product-list-filter/product-list-filter.js:30]: no magic string

Comment (M1ndForce) [reply src/pages/product-list-page/product-list-filter/product-list-filter.js:30]: it does not change the query in filters

**Review by jul4ajka — APPROVED** [review]

**Review by AndriyLymych — APPROVED** [review]

**Pull request merged**